tests/rptest/tests/rpk_tuner_test.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2022 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from rptest.services.cluster import cluster
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.clients.rpk_remote import RpkRemoteTool
+
+
+class RpkTunerTest(RedpandaTest):
+    def __init__(self, ctx):
+        super(RpkTunerTest, self).__init__(test_context=ctx)
+        self._ctx = ctx
+
+    @cluster(num_nodes=1)
+    def test_tune_prod_all(self):
+        """
+        Test will set production mode and execute rpk redpanda tune all,
+        we expect the command to exit with 1 if an error happens.
+        """
+        node = self.redpanda.nodes[0]
+        rpk = RpkRemoteTool(self.redpanda, node)
+        rpk.mode_set("prod")
+
+        rpk.tune("all")
+
+    @cluster(num_nodes=1)
+    def test_tune_fstrim(self):
+        """
+        Validate fstrim tuner execution,
+        fstrim was disabled in production mode https://github.com/redpanda-data/redpanda/issues/3068 
+        """
+        node = self.redpanda.nodes[0]
+        rpk = RpkRemoteTool(self.redpanda, node)
```

Comment:
Can these tests check `rpk.tune_fstrim`, and below, `rpk.transparent_hugepages` are false before swapping to true?

This will avoid the (not problematic, but redundant) scenario if THP or fstrim are set to true in mode prod: tune all will check these, so we'll be able to delete these tests.

Reply:
(not sure how compelling this comment/question is so feel free to dismiss)

Reply:
That's true, we also catch this change in `test_config_change_mode_prod`

Reply:
Right, I was thinking that if one of those is set to true, a person wont know to go into this file and remove these redundant tests